can/io/blf.py

```diff
@@ -220,6 +220,14 @@ def _parse_data(self, data):
         # Loop until a struct unpack raises an exception
         while True:
             self._pos = pos
+            #fix padding
+            if pos + obj_header_base_size > max_pos:
+                # This object continues in the next container
+                return
+            for i in range(4):
+                if data[pos+i:pos+i+4] == b"LOBJ":
+                    break
+            pos += i
```

Comment:
I would suggest using the `index` method instead since it is implemented in optimized C. It should also handle the case when LOBJ is split up in two containers.

```suggestion
            # Find next object after padding (depends on object type)
            try:
                pos = data.index(b"LOBJ", pos, pos + 8)
            except ValueError:
                if pos + 8 > max_pos:
                    # Not enough data in container
                    return
                raise BLFParseError("Could not find next object")
```

You would then also need to change line 178 to remove `memoryview` but that does not seem affect performance much.